openwisp_controller/config/whois/service.py

```diff
@@ -72,6 +72,19 @@ def is_whois_enabled(self):
             )
         return getattr(org_settings, "whois_enabled", app_settings.WHOIS_ENABLED)
 
+    @property
+    def is_estimated_location_enabled(self):
+        """
+        Check if the Estimated location feature is enabled.
+        This does not require to set cache as `is_whois_enabled` already sets it
```

Comment:
Do we have a test case for this?

Reply:
I will update the existing test case which asserts `cache.get` to assert `.set` is not called